src/ORM/Table.php

```diff
@@ -2490,6 +2490,11 @@ public function newEntities(array $data, array $options = [])
      *
      * You can use the `Model.beforeMarshal` event to modify request data
      * before it is converted into entities.
+     *
+     * When patching scalar values (null/booleans/string/integer/float), if the property
+     * presently has an identical value, the setter will not be called, and the
+     * property will not be marked as dirty. This is an optimization to prevent unnecessary field
+     * updates when persisting entities.
```

Comment:
... maybe add something like:
"If you want to run setters, either run them manually or mark fields dirty explicitly, use `EntityInferface::setDirty()`"?

Reply:
Why? I was aiming to document that on no-op updates properties are not dirtied.

Reply:
Because custom setters could do more than a no-op. But yes, well designed a single field no-change should have no side effects (similar to idempotency) - so pls ignore my comment.